easybuild/easyconfigs/g/GEMMA/GEMMA-0.97-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = 'GEMMA'
+version = '0.97'
+
+homepage = 'https://github.com/genetics-statistics/GEMMA'
+description = "Genome-wide Efficient Mixed Model Association"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/genetics-statistics/GEMMA/archive/']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('GSL', '2.3'),
+    ('Eigen', '3.3.1', '', True),
```

Comment:
@pescobar This should probably be bumped to the latest `Eigen` we have, and it's most likely also a build-only dep (since it's just header files)